dev/src/index.ts

```diff
@@ -677,7 +678,7 @@ export class Firestore {
    * });
    * bulkWriter.close();
    */
-  bulkWriter(options?: BulkWriterOptions): BulkWriter {
+  _bulkWriter(options?: BulkWriterOptions): BulkWriter {
```

Comment:
Make sure to also remove the `export { BulkWriter }` line.

Reply:
done.